src/Sarif/Baseline/ResultMatching/SarifLogResultMatcher.cs

```diff
@@ -94,9 +97,10 @@ public IEnumerable<SarifLog> Match(IEnumerable<SarifLog> previousLogs, IEnumerab
                 {
                     continue;
                 }
+
                 foreach (Run run in sarifLog.Runs)
                 {
-                    string toolName = run.Tool.Driver.Name;
+                    string toolName = run.Tool.Driver.Name.ToLower();
```

Comment:
>ToLower() [](http://example.com/codeflow?start=59&length=9)

this will make all tools lower case

Reply:
Don't do this! :) It is never a good idea to transform data when you are an API. Instead, please look for ways to make your comparisons case-insensitive when necessary. For example, create a hash set with a case-insensitive comparer (for the set members).

By the way, in the rare cases when ToLower() is required, you should always prefer ToLowerInvariant() in order to avoid culture-sensitive issues (for letters that don't round-trip properly). Search the web for 'turkish i problem'.

Reply:
got it.